### PR TITLE
Update make-calls.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Added mobile content to calls docs.

Addresses: https://github.com/mattermost/mattermost-mobile/pull/6781
Addresses: https://github.com/mattermost/mattermost-mobile/pull/6979

View the changes here: http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6232/channels/make-calls.html

### CODE REVIEW

File: source/channels/make-calls.rst [hunk 1]
```diff
@@ -28,16 +27,59 @@ Starting from Mattermost v7.7 you can:
...
+
+  .. tab:: Mobile
+  
+    To do this, expand the call window using the arrows in the top-right of the active call banner. The select **More > Call Thread**.
+    
+Record a call
+-------------
 
-From Mattermost v7.7 if you're the host of a meeting, you can record the call. 
+From Mattermost v7.7 if you're the host of a meeting, you can record the call.
```
Comment (cwarnermm) [inline source/channels/make-calls.rst:82]: Does it make sense to mention that call recording isn't supported via mobile yet/at all?

File: source/channels/make-calls.rst [hunk 2]
```diff
@@ -28,16 +27,59 @@ Starting from Mattermost v7.7 you can:
...
+Emojis
+------
+
+You can use emojis to react during a call.
+
+.. tabs::
+
+  .. tab:: Desktop
+
+    To do this, expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.
```
Comment (cwarnermm) [inline source/channels/make-calls.rst:58]: is the "to do this" necessary? What are your thoughts on starting each sentence with a verb instead?

**Review by cwarnermm — APPROVED** [review]
Provided non-blocking feedback as questions inline. Approving for expediency

File: source/channels/make-calls.rst [hunk 3]
```diff
@@ -28,16 +27,59 @@ Starting from Mattermost v7.7 you can:
...
 
-To start a call, select **Start call** in the channel header. Any active team member in the channel can join a call, whether it's a public or private channel. If someone from outside of the organization wants to join a call, you'll need to provide them with a guest account, and add them to the channel. Users who are archived or not registered can't join a call.
+Start a call
+------------
+
+.. tabs::
+
+  .. tab:: Desktop
+
+    To start a call, select **Start call** in the channel or message header. When you start a call in a channel, you're muted by default. In a direct or group message you're unmuted by default.
```
Comment (streamer45) [inline source/channels/make-calls.rst:37]: For completeness I should mention that `/call start` or `/call join` slash commands can be used as well. Or even easier through the keyboard shortcut. That said, I am okay if we only want to keep the most common way here.

File: source/channels/make-calls.rst [hunk 4]
```diff
@@ -28,16 +27,59 @@ Starting from Mattermost v7.7 you can:
...
+    To do this, expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.
+
+  .. tab:: Mobile
+  
+    To do this, expand the call window using the arrows in the top-right of the active call banner. From there, select **React**.
+      
+Chat in a call
+--------------
+
+You can create a chat thread during a call, which is associated with that call.
```
Comment (streamer45) [inline source/channels/make-calls.rst:67]: A chat thread is created automatically for every new call. What the user can do is opening it up from the expanded window and interact same as on Mattermost.

File: source/channels/make-calls.rst (hunk 1 above)
Comment (streamer45) [reply source/channels/make-calls.rst:82]: Managing recordings (starting/stopping) is supported from mobile as well. Unless I am misunderstanding the concern.

**Review by streamer45 — COMMENTED** [review]
Thanks @justinegeffen , just left a couple of minor comments.

File: source/channels/make-calls.rst [hunk 5]
```diff
@@ -28,16 +27,59 @@ Starting from Mattermost v7.7 you can:
...
+
+.. tabs::
+
+  .. tab:: Desktop
+
+    To start a call, select **Start call** in the channel or message header. When you start a call in a channel, you're muted by default. In a direct or group message you're unmuted by default.
+
+  .. tab:: Mobile
+     
+     To start a call, select the three horizontal dots at the top right-corner of your device. Then select **Start Call**. The call starts in speaker mode by default.
```
Comment (cpoile) [inline source/channels/make-calls.rst:41]: You can also start a call by clicking the Channel Info menu (the channel title) and selecting Start Call from there.

**Review by cpoile — APPROVED** [review]
Thanks Justine. Looks good (+ Claudio's notes)

File: source/channels/make-calls.rst (hunk 5 above)
Comment (justinegeffen) [inline source/channels/make-calls.rst:41]:
```suggestion
     To start a call, select the channel info menu. Then select **Start Call**. The call starts in speaker mode by default.
```

File: source/channels/make-calls.rst (hunk 2 above)
Comment (justinegeffen) [inline source/channels/make-calls.rst:58]:
```suggestion
    Expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.
```

File: source/channels/make-calls.rst [hunk 6]
```diff
@@ -28,16 +27,59 @@ Starting from Mattermost v7.7 you can:
...
+    Expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.
+
+  .. tab:: Mobile
+  
+    To do this, expand the call window using the arrows in the top-right of the active call banner. From there, select **React**.
+      
+Chat in a call
+--------------
+
+You can create a chat thread during a call, which is associated with that call.
```
Comment (justinegeffen) [inline source/channels/make-calls.rst:67]:
```suggestion
A chat thread is created automatically for every new call.
```

File: source/channels/make-calls.rst [hunk 7]
```diff
@@ -28,16 +27,59 @@ Starting from Mattermost v7.7 you can:
...
+
+.. tabs::
+
+  .. tab:: Desktop
+
+    Expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.
+
+  .. tab:: Mobile
+  
+    To do this, expand the call window using the arrows in the top-right of the active call banner. From there, select **React**.
```
Comment (justinegeffen) [inline source/channels/make-calls.rst:62]:
```suggestion
    Expand the call window using the arrows in the top-right of the active call banner. From there, select **React**.
```

File: source/channels/make-calls.rst (hunk 1 above)
Comment (streamer45) [reply source/channels/make-calls.rst:82]: @justinegeffen It should be under the dot menu (More option) in the call screen:

image

Comment (justinegeffen) [reply source/channels/make-calls.rst:82]: I don't have that in my version - this is from a direct message call. 

Screenshot 2023-02-22 at 21 47 48

Comment (justinegeffen) [reply source/channels/make-calls.rst:82]: Using iOS 16

Comment (streamer45) [reply source/channels/make-calls.rst:82]: @justinegeffen Are you trying on a recordings enabled server? (e.g. https://calls.test.mattermost.cloud/). Nothing will show unless the feature is enabled and active.

Comment (justinegeffen) [reply source/channels/make-calls.rst:82]: Oh good question. This is on community, staff team. Not sure if we have it enabled there?

Comment (streamer45) [reply source/channels/make-calls.rst:82]: Nope, Community doesn't have the "official" feature enabled so it won't appear there. You can give the cloud test I linked above a try.

Comment (justinegeffen) [reply source/channels/make-calls.rst:82]: Brilliant, thank you!

**Pull request merged**